eid/id.go

```diff
@@ -1,10 +1,30 @@
 package eid
 
 import (
+	"github.com/oklog/ulid"
 	"github.com/teris-io/shortid"
+	"math/rand"
+	"time"
 )
 
+var entropyC = make(chan string, 100)
+
+func init() {
+	go func() {
+		source := ulid.Monotonic(rand.New(rand.NewSource(time.Now().UnixNano())), 0)
```

Comment:
This library is interesting. It doesn't use lower case in the documentation example and states that its alphabet is `0123456789ABCDEFGHJKMNPQRSTVWXYZ` and says that `I, L, O and U are excluded` to avoid confusion. However, all uppercase letters are hard to read.

Also, these ids appear to be longer than what `shortid` provided. Is there a possibility of using a larger encoding alphabet (i.e. base64) to have shorter string representations?